includes/js/wp-performance-wizard.js

```
@@ -153,24 +203,65 @@
 					step++
-					echoToTerminal( '<br><div class="info-chip user-chip">USER</div><br>' );
+					echoToTerminal( '<br><div class="info-chip user-chip" aria-label="User">USER</div><br>' );
 					echoToTerminal( '<div class="k"><br>' + promptForDisplay  + '</div>' );
-					result = await runPerfomanceWizardPrompt( nextStep.user_prompt, step, false, selectedModel );
-					echoToTerminal();
-					echoToTerminal( result );
+
+					// Show spinner while AI processes the prompt
+					showSpinner( 'AI is analyzing and generating response...' );
+
+					let result;
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)